### ISSUE TITLE
Empty type field in VariableDefinition

### ISSUE THREAD

**Issue by kpeters-cbsi** [apollographql/apollo-rs#138]
## Description

For a given `apollo_parser::ast::VariableDefinition` `v`, calling `v.ty()` returns None.

### Steps to reproduce

Execute the following program:

```rust
use apollo_parser::Parser;
const QUERY: &str = r#"
query MatchupAnalysisQuery(
    $homeTeamAbbrev: String!
    $awayTeamAbbrev: String!
    $sportType: GameSportTypeEnumType!
    $poolId: ID
    $weekNumber: Int
    $gameInstanceUid: String
  ) {
    matchupAnalysis(
      homeTeamAbbrev: $homeTeamAbbrev
      awayTeamAbbrev: $awayTeamAbbrev
      sportType: $sportType
      poolId: $poolId
      weekNumber: $weekNumber
      gameInstanceUid: $gameInstanceUid
    ) {
      id
    }
}"#;
fn main() {
    let parser = Parser::new(QUERY);
    let ast = parser.parse();
    assert!(&ast.errors().is_empty());
    let doc = ast.document();
    for def in doc.definitions() {
        match def {
            apollo_parser::ast::Definition::OperationDefinition(operation) => {
                let vars = operation.variable_definitions().unwrap();
                let fmt = format!("{}", &vars);
                dbg!(&vars, fmt);
                for var in vars.variable_definitions().into_iter() {
                    let name = var.variable().unwrap().name().unwrap().to_string();
                    let ty = match var.ty() {
                        Some(ty) => ty.to_string(),
                        None => "NO TYPE".to_owned()
                    };
                    println!("{}: {}", name, ty);
                }
            },
            _ => unimplemented!()
        }
    }
}

```
### Expected result

Output:

```
homeTeamAbbrev: String!
awayTeamAbbrev: String!
sportType: GameSportEnumType!
poolId: ID
weekNumber: Int
gameInstanceUid: String
```

### Actual result

Output:

```
homeTeamAbbrev: NO TYPE
awayTeamAbbrev: NO TYPE
sportType: NO TYPE
poolId: NO TYPE
weekNumber: NO TYPE
gameInstanceUid: NO TYPE
```

### Environment

- Operating system and version: MacOS BigSur Version 11.6
- Shell (bash/zsh/powershell): zsh
- `apollo-rs` crate: ??
- Crate version: `apollo-parser` 0.1.0

**Comment by lrlna** [comment]
@kpeters-cbsi ah! let me publish a new version for you -- we fixed this issue in #127, but it just hasn't been published.

**Comment by lrlna** [comment]
@kpeters-cbsi I just published a new version, `0.2.0`. It has a fix for the problem you're running into above.

The release comes with a breaking change - `ast.errors()` now returns an Iterator - so you will have to change your errors assert to `assert!(ast.errors().len(), 0)`. You can read up on it in more detail in the [Changelog](https://github.com/apollographql/apollo-rs/blob/main/crates/apollo-parser/CHANGELOG.md#020---2021-12-22).

Let me know if you're still running into trouble!
